apps/web/src/pages/index.tsx

```diff
@@ -1,9 +1,18 @@
 import type { NextPage } from 'next';
 import 'reflect-metadata';
-import { Sidebar } from '../components/Sidebar';
+import { Form, Header, PageTitle, Sidebar } from '@components';
 
 const Home: NextPage = () => {
-  return <Sidebar></Sidebar>;
+  return <>
+    <div className="flex overflow-x-hidden h-screen mr-auto">
+      <Sidebar />
+      <div className="w-full p-4 flex-col">
+        <Header />
+        <PageTitle title="Resource Planning" description="Planning resourcing plan based on skills and tasks within your unit."/>
+        <Form />
```

Comment:
I think this should be displayed only if they click on Planning btn

Reply:
I think this was not in the task. Probably has to do with side nav. But I can fix this once this branch is merged